src/utils/dataSubmissionUtils.ts

```diff
@@ -1,3 +1,4 @@
+import { ADMIN_OVERRIDE_CONDITIONS, SUBMIT_BUTTON_CONDITIONS } from "../config/SubmitButtonConfig";
 import { safeParse } from "./jsonUtils";
 
 export type SubmitInfo = {
```

Comment:
TBD: The comment isn't appearing

Reply:
Thanks for catching this! Removed in [4857cf1](https://github.com/CBIIT/crdc-datahub-ui/pull/464/commits/4857cf1f9c987a538758c2e37cd5362d8165e930).